### PR TITLE
Story/130376 task 131103 update first time sign in flow

### PR DESCRIPTION
**Pull request by diogo-costa-bjss**

### CODE REVIEW

File: src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts [hunk 1]
```diff
@@ -4,33 +4,38 @@ import { RouterTestingModule } from '@angular/router/testing';
...
 import { AppInjector, CoreModule } from '@modules/core';
 import { InnovatorModule } from '@modules/feature-modules/innovator/innovator.module';
 import { AuthenticationStore, StoresModule } from '@modules/stores';
 
-import { FirstTimeSigninInnovationNewComponent } from './innovation-new.component';
+import { FirstTimeSigninComponent } from './first-time-signin.component';
 
 import { InnovatorService } from '../../services/innovator.service';
 
+<<<<<<< HEAD:src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts
```
Comment (kduarte-bjss) [inline src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts:16]: merge message

File: src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts [hunk 2]
```diff
@@ -4,33 +4,38 @@ import { RouterTestingModule } from '@angular/router/testing';
...
 
-describe('FeatureModules/Innovator/Pages/FirstTimeSignin/FirstTimeSigninInnovationNewComponent', () => {
+describe('FeatureModules/Innovator/Pages/FirstTimeSignin/FirstTimeSigninComponent', () => {
 
   let router: Router;
   let routerSpy: jest.SpyInstance;
 
   let authenticationStore: AuthenticationStore;
   let innovatorService: InnovatorService;
+<<<<<<< HEAD:src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts
```
Comment (kduarte-bjss) [inline src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts:31]: same as above

File: src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts (hunk 1 above)
Comment (diogo-costa-bjss) [reply src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts]: Fixed in last commit

File: src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts (hunk 2 above)
Comment (diogo-costa-bjss) [reply src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.component.spec.ts]: Fixed in last commit

**Review by manuelsousa-bjss — APPROVED** [review]

**Review by kduarte-bjss — APPROVED** [review]

**Pull request merged**